### PR TITLE
Old PostgreSQL version error explaination in the FAQ.

### PR DESCRIPTION
**Pull request by Natim**

### CODE REVIEW

**Comment by phrawzty** [comment]
While this answer may be technically correct, it leaves me unsatisfied:
* What is generating this error?
* When might this error message be seen?
* Can the problem be ignored?
* Why does upgrading to pgsql 9.4+ solve the problem?
* Does anything else need to be upgraded?

etc.

**Comment by Natim** [comment]
- What is generating this error? **The PostgreSQL driver that cannot activate the JSONbin functionnality not present with old server version.**
- When might this error message be seen? **When using a PostgreSQL older than 9.4**
- Can the problem be ignored? **Nope the server doesn't run in that case and fails with this error**
- Why does upgrading to pgsql 9.4+ solve the problem? **Because after PostgreSQL9.4 server have got JSONBin support**
- Does anything else need to be upgraded? **No**

**Comment by Natim** [comment]
@phrawzty Based on this new information how shall I rephrase the FAQ answer?

**Comment by leplatrem** [comment]
Also, ideally I wouldn't mix *troubleshooting* and overview questions.

**Comment by Natim** [comment]
>  I wouldn't mix troubleshooting and overview questions.

Yes good point, shall I create troubleshooting file?

**Comment by phrawzty** [comment]
If this is really a FAQ it could link to the (hypothetical) troubleshooting page.

**Comment by Natim** [comment]
Updated with your remarks.

**Comment by Natim** [comment]
Updated with Dan rephrasing. Thanks :+1:

File: docs/troubleshooting.rst [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+###############
+
+.. _troubleshooting:
+
+We are doing the best we can so you do not have to read this section.
+
+In case you do, please find the common problems you may have below.
+
+If you do not find your problem here, please file an issue or ask for
+help on `#storage <http://chat.mibbit.com/?server=irc.mozilla.org&channel=#storage>`_
```
Comment (leplatrem) [inline docs/troubleshooting.rst]: I would prefer a link to communication channels http://kinto.readthedocs.org/en/latest/contributing.html#communication-channels

File: docs/troubleshooting.rst [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+In case you do, please find the common problems you may have below.
+
+If you do not find a solution to your problem here, please
+:ref:`ask for help <communication_channels>`!
+
+
+Module object has no attribute 'register_json'
+==============================================
+
+Kinto uses PostgreSQL ``JSONBin`` feature, which allow Kinto to
```
Comment (phrawzty) [inline docs/troubleshooting.rst]: Kinto uses *the* JSONBin feature of PostgreSQL, which is used to store native JSON objects efficiently. Support for this feature was added in PostgreSQL 9.4.

File: docs/overview.rst [hunk 3]
```diff
@@ -222,3 +222,10 @@ Cliquet is a toolkit for designing micro-services. Kinto is a server built
 using that toolkit.
 
 `Read more (in french) about the differences <http://www.servicedenuages.fr/pourquoi-cliquet>`_.
+
+
+When running Kinto I have got an exception, what's wrong?
```
Comment (phrawzty) [inline docs/overview.rst]: I'm seeing an Exception error - what's wrong?

File: docs/troubleshooting.rst [hunk 4]
```diff
@@ -0,0 +1,23 @@
+Troubleshooting
+###############
+
+.. _troubleshooting:
+
+We are doing the best we can so you do not have to read this section.
+
+In case you do, please find the common problems you may have below.
```
Comment (phrawzty) [inline docs/troubleshooting.rst]: That said, we've included solutions (or at least explanations) for some common problems below.

**Comment by phrawzty** [comment]
`r+`

File: docs/troubleshooting.rst [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+If you do not find a solution to your problem here, please
+:ref:`ask for help <communication_channels>`!
+
+
+Module object has no attribute 'register_json'
+==============================================
+
+Kinto uses the ``JSONBin`` feature of PostgreSQL, which is used to
+store native ``JSON objects`` efficiently. Support for this feature
+wasa added in PostgreSQL 9.4.
```
Comment (jvehent) [inline docs/troubleshooting.rst]: [wazzaaa](https://www.youtube.com/watch?v=8ejcKyRAHAg)

**Pull request merged**